### PR TITLE
Remove sidebar actions slot max height

### PR DESCRIPTION
**Pull request by ma12-co**
Needed to fit talk's description in the header slot.

Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

**Review by skjnldsv — COMMENTED** [review]
Maybe we should even rename the slot :p

**Comment by ma12-co** [comment]
How would you like it named?? :)

**Comment by skjnldsv** [comment]
> How would you like it named?? :)

No clue, could be description. I am not the best to find names :see_no_evil:

File: src/components/AppSidebar/AppSidebar.vue [hunk 1]
```diff
@@ -215,8 +215,8 @@
 						</Actions>
 					</div>
 				</div>
-				<div v-if="$slots['primary-actions'] && !empty" class="app-sidebar-header__action">
-					<slot name="primary-actions" />
+				<div v-if="$slots['description'] && !empty" class="app-sidebar-header__action">
```
Comment (skjnldsv) [inline src/components/AppSidebar/AppSidebar.vue:218]:
```suggestion
				<div v-if="$slots['description'] && !empty" class="app-sidebar-header__description">
```

File: src/components/AppSidebar/AppSidebar.vue [hunk 2]
```diff
@@ -692,7 +692,6 @@ $top-buttons-spacing: 6px;
 		&__action {
```
Comment (skjnldsv) [inline src/components/AppSidebar/AppSidebar.vue:692]:
```suggestion
		&__description {
```

File: src/components/AppSidebar/AppSidebar.vue (hunk 1 above)
Comment (skjnldsv) [reply src/components/AppSidebar/AppSidebar.vue:218]: Ah right, we already have app-sidebar-header__desc
Maybe a better name then?

`app-sidebar-header__action` => `app-sidebar-header__description`
`app-sidebar-header__desc` => `app-sidebar-header__details`

?

**Review by skjnldsv — COMMENTED** [review]
In that case, since we're at it with the breaking changes, rename `secondary-actions` to  `actions` ? As well as `tertiary-actions` to `star` ? :thinking: :shrug: 

cc @raimund-schluessler

**Comment by ma12-co** [comment]
I just wanted to remove a css rule though :sweat_smile:

**Comment by raimund-schluessler** [comment]
> 
> 
> I just wanted to remove a css rule though 😅

Would this be a breaking change as well?

Renaming these slots requires every app using the appsidebar to adjust (I don't mind though, I don't use it yet, just thinking loud 🙈)

> As well as `tertiary-actions` to `star` ?

It's only a star by default, but since it's a slot, it can be anything. So `star` might be to specific as a slot name.

**Comment by ma12-co** [comment]
I'm fine with anything here, just lmk what to do :)

**Comment by skjnldsv** [comment]
> Would this be a breaking change as well?

There is another pr that touches the sidebar that is breaking change yes :)

**Comment by skjnldsv** [comment]
> It's only a star by default, but since it's a slot, it can be anything. So `star` might be to specific as a slot name.

I would argue that 'tertiary-actions' isn't really self-explanatory either :stuck_out_tongue:

**Comment by ma12-co** [comment]
So should I do this?? https://github.com/nextcloud/nextcloud-vue/pull/1566#pullrequestreview-528155537

**Comment by ma12-co** [comment]
I got back to renaming only the slot and classes referring to that slot because of naming conflicts. Let's address the rest of the names in separate PRs if needed.

**Review by skjnldsv — APPROVED** [review]

**Review by Ivansss — APPROVED** [review]

**Pull request merged**